test-files/cases/hoot-js/poi/poi-generic-33/Expected.osm

```diff
@@ -1,24 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<osm version="0.6" generator="hootenanny" srs="GEOGCS[&quot;WGS 84&quot;,&#10;    DATUM[&quot;WGS_1984&quot;,&#10;        SPHEROID[&quot;WGS 84&quot;,6378137,298.257223563,&#10;            AUTHORITY[&quot;EPSG&quot;,&quot;7030&quot;]],&#10;        TOWGS84[0,0,0,0,0,0,0],&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;6326&quot;]],&#10;    PRIMEM[&quot;Greenwich&quot;,0,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;8901&quot;]],&#10;    UNIT[&quot;degree&quot;,0.0174532925199433,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;9108&quot;]],&#10;    AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]">
-    <node visible="true" id="-3" timestamp="1970-01-01T00:00:00Z" version="1" lat="2.8672985999999998" lon="22.3670612999999996">
+<osm version="0.6" generator="hootenanny" srs="+epsg:4326">
+    <bounds minlat="2.255955" minlon="22.3062313" maxlat="2.8672986" maxlon="22.9900745"/>
+    <node visible="true" id="-340017" timestamp="1970-01-01T00:00:00Z" version="1" lat="2.2559549999999997" lon="22.9900744999999986">
         <tag k="use" v="residential"/>
         <tag k="place" v="village"/>
-        <tag k="name" v="Place2"/>
+        <tag k="name" v="somewhere"/>
```

Comment:
This change is actually an ordering issue, it matches with the third node below.